stubs/setuptools/setuptools/_distutils/dist.pyi

```diff
@@ -54,7 +57,84 @@ class DistributionMetadata:
 class Distribution:
     cmdclass: dict[str, type[Command]]
     metadata: DistributionMetadata
-    def __init__(self, attrs: Mapping[str, Incomplete] | None = ...) -> None: ...
+    def __init__(self, attrs: Mapping[str, Any] | None = ...) -> None: ...
```

Comment:
I don't think it's possible to know all attrs, so I _think_ this is Any, not Incomplete.